src/components/ebay-carousel/README.md

```diff
@@ -26,7 +27,8 @@ Name | Type | Stateful | Required | Description
 
 Name | Type | Stateful | Required | Description
 --- | --- | --- | --- | ---
-`a11y-status-text` | String | No | Yes | status text (default: "Showing Slide {currentSlide} of {totalSlides} - Carousel")
+`a11y-heading-text` | String | No | No | a11y heading text used in combination with `a11y-status-text` (default: "{heading} Showing Slide {currentSlide} of {totalSlides} - Carousel") _**Note**: a `-` will be appended to the heading automatically. There's no need for the developer or model data to add it._
```

Comment:
Name and description needs updating from 'heading' to 'title'.

Reply:
Ah, I thought that I had done this! (I think I know what happened, though.) I'll get that fixed.